great_expectations/core/expectation_validation_result.py

```diff
@@ -657,3 +660,13 @@ def make_expectation_suite_validation_result(self, data, **kwargs):
 
 expectationSuiteValidationResultSchema = ExpectationSuiteValidationResultSchema()
 expectationValidationResultSchema = ExpectationValidationResultSchema()
+
+
+def _make_serializable(data: Any) -> Any:
```

Comment:
This is obviously not the most robust implementation, but it gets things passing. I think it's best to consider this a work in progress that can be updated as needed.